documentation/modules/metrics/con-metrics-cruise-control.adoc

```diff
@@ -0,0 +1,85 @@
+You can use your own configuration or an example file provided by Strimzi called `kafka-cruise-control-metrics.yaml`.
+
+You add the configuration to the `metricsConfig` of the `CruiseControl` property in the `Kafka` resource.
+The configuration enables JMX for Cruise Control and uses the {JMXExporter} to expose Cruise Control metrics through an HTTP endpoint.
+The HTTP endpoint is scraped by the Prometheus server.
+
+.Example metrics configuration for Cruise Control
+[source,yaml,subs="+quotes,attributes"]
+----
+  apiVersion: {KafkaApiVersion}
+  kind: Kafka
+  metadata:
+    name: my-cluster
+  Spec:
+    # ...
+    cruiseControl:
+      # ...
+      metricsConfig:
+        type: jmxPrometheusExporter
+        valueFrom:
+          configMapKeyRef:
+            name: cruise-control-metrics
+            key: metrics-config.yml
+  ---
+  kind: ConfigMap
+  apiVersion: v1
+  metadata:
+    name: cruise-control-metrics
+    labels:
+      app: strimzi
+  data:
+    metrics-config.yml: |
+    # _metrics configuration..._
+----
+
+== Viewing Cruise Control metrics
+
+Having exposed Cruise Control metrics, you can use a monitoring tool to view the information on the data.
+Strimzi provides an xref:assembly-metrics-config-files-str[example Grafana dashboard] to provide visualizations of Cruise Control metrics.
+The dashboard is a JSON file called `strimzi-cruise-control.json`.
+The exposed metrics provide the monitoring data when you xref:proc-metrics-grafana-dashboard-str[enable the Grafana dashboard].
+
+=== Monitoring Balancedness scores
+
+Cruise Control metrics include balancedness scores.
+Balancedness is the measure of how evenly a workload is distributed in a Kafka cluster.
+
+The Cruise Control metric for balancedness score (`balancedness-score`) differs from the balancedness score in the `KafkaRebalance` resource.
+Cruise Control calculates each score using different goals.
+The former uses `anomaly.detection.goals`, while the latter uses `default.goals`.
+The goals are specified in the `spec.cruiseControl.config` of the `Kafka` custom resource.
+
+NOTE: Refreshing a `KafkaRebalance` resource fetches the latest state of a Kafka cluster.
+The refresh also requests an optimization proposal from Cruise Control. Continual refreshes can impact performance monitoring in certain circumstances.
+For example, the `goals` configured in the `KafkaRebalance` resource might differ from the `default.goals` used by Cruise Control.
+
+=== Alerts on anomaly detection
+
+Cruise Control’s anomaly detection is enabled for use with AMQ Streams.
+You can use anomaly detection when monitoring Cruise Control metrics.
+Cruise Control’s anomaly notifier provides alerts on broker failures and other conditions that block the generation of optimization goals, as well as any actions taken.
```

Comment:
alerts -> data

Following Jakub's comment on what exactly creates/sends "alerts". Cruise Control anomaly notifier provides metrics/data for an external alertmanager to alerts on

Reply:
Changed to:

> Cruise Control’s anomaly notifier provides metrics data for conditions that block the generation of optimization goals, such as broker failures. An alerting tool like Prometheus Alertmanager can use these metrics to send notifications about the events.